src/legacy/core_plugins/kibana/common/field_formats/types/static_lookup.ts

```diff
@@ -17,32 +17,46 @@
  * under the License.
  */
 
-function convertLookupEntriesToMap(lookupEntries) {
-  return lookupEntries.reduce((lookupMap, lookupEntry) => {
+import {
+  FieldFormat,
+  TEXT_CONTEXT_TYPE,
+  KBN_FIELD_TYPES,
+  FieldFormatConvert,
+} from '../../../../../../plugins/data/common/';
+
+function convertLookupEntriesToMap(lookupEntries: any) {
+  return lookupEntries.reduce((lookupMap: any, lookupEntry: any) => {
     lookupMap[lookupEntry.key] = lookupEntry.value;
     return lookupMap;
   }, {});
 }
 
-export function createStaticLookupFormat(FieldFormat) {
+export function createStaticLookupFormat() {
```

Comment:
I see that now we execute **create*Format** methods without any arguments. I think we can simplify logic and return class without this extra wrapper

@ppisljar  what do you think?  All what we need to do here just a modify a little src\src\legacy\ui\public\registry\field_formats.js file

Reply:
++

Reply:
actually, that registry should be removed
also we need to fix the server side registry.

for this PR i think this is out of scope, lets leave these as functions and still use the ui/registry
in the next PR we can remove the registry and we'll also remove this function wrappers.

Reply:
https://github.com/elastic/kibana/issues/47440